cinema/tests/test_movie_api.py

```diff
@@ -111,9 +103,7 @@
         self.assertEqual(response.data["genres"][1]["name"], "Comedy")
         self.assertEqual(response.data["actors"][0]["first_name"], "Kate")
         self.assertEqual(response.data["actors"][0]["last_name"], "Winslet")
-        self.assertEqual(
-            response.data["actors"][0]["full_name"], "Kate Winslet"
-        )
+        self.assertEqual(response.data["actors"][0]["full_name"], "Kate Winslet")
```

Comment:
This assertion verifies that the actor's full name is correctly retrieved, ensuring the API returns the expected data structure.